contracts/test/mocks/TokenReturnMissingMock.sol

```diff
@@ -0,0 +1,105 @@
+// Non-standards compliant token that is missing return values for
+// `transfer()`, `transferFrom()`, and `approve().
+// Modified from https://github.com/OpenZeppelin/openzeppelin-solidity/blob/a9f910d34f0ab33a1ae5e714f69f9596a02b4d91/contracts/token/ERC20/StandardToken.sol
+
+pragma solidity 0.4.24;
+
+import "../../lib/math/SafeMath.sol";
+
+
+contract TokenReturnMissingMock {
```

Comment:
Maybe inherit from the `GeneralERC20` interface?

Reply:
Don't think this is necessary, especially as the other token mocks wouldn't inherit from it.